src/main.ts

```diff
@@ -1,22 +1,35 @@
-import { Plugin } from 'obsidian';
-import { APP_NAMING, BibleReferencePluginSettings, DEFAULT_SETTINGS } from './data/constants';
-import { BibleReferenceSettingTab } from './ui/BibleReferenceSettingTab';
-import { VerseEditorSuggestor } from './VerseEditorSuggestor';
-
+import { Plugin } from "obsidian";
+import {
+  APP_NAMING,
+  BibleReferencePluginSettings,
+  DEFAULT_SETTINGS,
+} from "./data/constants";
+import { BibleReferenceSettingTab } from "./ui/BibleReferenceSettingTab";
+import { VerseEditorSuggestor } from "./suggesetor/VerseEditorSuggestor";
+import { VerseSuggestModal } from "./suggesetor/VerseSuggestModal";
 
 export default class BibleReferencePlugin extends Plugin {
   settings: BibleReferencePluginSettings;
+  suggestModal: VerseSuggestModal;
 
   async onload() {
-    console.log('loading plugin -', APP_NAMING.appName);
+    console.log("loading plugin -", APP_NAMING.appName);
 
     await this.loadSettings();
+    this.suggestModal = new VerseSuggestModal(this.app, this.settings);
     this.addSettingTab(new BibleReferenceSettingTab(this.app, this));
     this.registerEditorSuggest(new VerseEditorSuggestor(this, this.settings));
+    this.addCommand({
+      id: "obr-lookup",
+      name: "Verse Lookup",
+      callback: () => {
+        this.suggestModal.open();
+      },
+    });
   }
 
   onunload() {
-    console.log('unloading plugin', APP_NAMING.appName);
+    console.log("unloading plugin", APP_NAMING.appName);
```

Comment:
I looks like prettier decided to reformat a few things. Let me know if you want these style changes reverted.